### DISCUSSION TITLE
请问dperf是否支持在arm64上编译测试，如果支持的话，在arm64上测试结果如何

### DISCUSSION THREAD

**Discussion by shipinsky** [baidu/dperf#418] [Q&A]
请问dperf是否支持在arm64上编译测试，如果支持的话，在arm64上测试结果如何

**Comment by pengjianzhang** [comment]
支持，性能与CPU、网卡相关，主页上的数据是ARM CPU上测出来的
如果喜欢dperf 请加个star 谢谢

**Reply by shipinsky** [reply]
谢谢，抽空在arm64上体验一把。已加star
